private/DeploymentState.class.ps1

```diff
@@ -54,30 +54,45 @@ function Get-StateFromService {
         [string] $StorageAccountName
     )
 
-        try {
-            $StorageContext = New-AzStorageContext -StorageAccountName $StorageAccountName -ErrorAction Stop
-            $StorageContainer = Get-AzStorageContainer -Name 'azure-synapse-tools' -Context $StorageContext -ErrorAction Stop
-            $DeploymentStateFile = $StorageContainer.CloudBlobContainer.GetBlockBlobReference("$($targetSynapse.name)_deployment_state.json")
-            $res = $DeploymentStateFile.DownloadText()
-            if ($res) {
-                $res = $res | ConvertFrom-Json
-            }
+    Try {
+        $StorageContext = New-AzStorageContext -StorageAccountName $StorageAccountName -ErrorAction Stop
+        $StorageContainer = Get-AzStorageContainer -Name 'azure-synapse-tools' -Context $StorageContext -ErrorAction Stop
+        $DeploymentStateFile = $StorageContainer.CloudBlobContainer.GetBlockBlobReference("$($targetSynapse.name)_deployment_state.json")
+    }
+    Catch {
+        throw $_.Exception
+    }
```

Comment:
Why catch an error and throw an exception? Why just do not remove the TRY/CATCH?